### ISSUE TITLE
Invalid signature file digest for Manifest main attributes

### ISSUE THREAD

**Issue by seras42** [Peergos/nabu#91]
Good day,
Whenever i try to use this implementation, including the nabu-chat example, i get this exception:
```Caused by: java.lang.SecurityException: Invalid signature file digest for Manifest main attributes```
Perhaps do you know what could cause it? Is this problem coming from my system or the project itself?

This is what i use for gradle:
```
repositories {
    mavenCentral()
    maven { url "https://jitpack.io"}
}

dependencies {
    implementation 'com.github.Peergos:nabu:latest.release'
}
```

**Comment by ianopolous** [comment]
I can't reproduce this. What JVM are you using?

**Comment by ianopolous** [comment]
Ok that should be fixed with v0.7.8

**Comment by seras42** [comment]
Confirmed. It works with v0.7.8
